### DISCUSSION TITLE
Cross-seed failed to inject, saving instead

### DISCUSSION THREAD

**Discussion by eski-dev** [cross-seed/cross-seed#507] [Q&A] [answered]
qbittorrent v4.5.5 (docker)
cross-seed v5.5.1 (docker)

I know there is a yet-to-be-completed Troubleshooting item [here](https://www.cross-seed.org/docs/reference/faq-troubleshooting#failed-to-inject-saving-instead), but I just set up cross-seed and am currently running into this issue.
Any torrent which contains a directory with files beneath it cross-seeds appropriately. Any single file torrents fail to inject and "save instead".

If possible, please respond with the resolution or I can help with testing if necessary to expedite the FAQ answer.

**Comment by zakkarry** [comment]
I'd recommend posting logs surrounding the issue or coming into the discord to have a discussion on something we can replicate.

With no information about your setup or the variables in it, it's kind of hard to give a solution, for me at least.

**Comment by zakkarry** [comment, marked as answer]
Resolved on Discord. torrentDir contained torrents not present in the client.
